providers/infrastructure-vsphere/v1.4.1/cconly/base.yaml

```diff
@@ -468,6 +468,12 @@ spec:
       openAPIV3Schema:
         type: object
         properties: {}
+  - name: controlPlaneTaint
+    required: false
+    schema:
+      openAPIV3Schema:
+        type: boolean
+        default: true
```

Comment:
why not wrap this under the controlPlane object and have it set node taints more generally? this is almost certainly going to be a requested feature.

Reply:
We wanted to make it flexible by the user, this is conditional patch and would remove taint only if the user sets this variable as false. We are setting this variable to true by default.

Reply:
> have it set node taints more generally
By this, did you mean accepting a list of taints from the user? Assuming that's what you meant - we decided to keep it a bool to make the experience of creating a single node cluster easier. 

That is fair feedback though. When we went through our TEP review, we opted for the boolean to limit scope. We could always enhance this in a future iteration to make this more generic. 

Does that make sense @tenczar?

Reply:
Yeah, that makes sense.